workers/unity/Packages/manifest.json

```diff
@@ -6,8 +6,8 @@
     "com.improbable.gdk.testutils": "file:com.improbable.gdk.testutils",
     "com.improbable.gdk.transformsynchronization": "file:com.improbable.gdk.transformsynchronization",
     "com.unity.burst": "0.2.4-preview.18",
-    "com.unity.entities": "0.0.12-preview.8",
-    "com.unity.incrementalcompiler": "0.0.42-preview.14",
+    "com.unity.entities": "0.0.12-preview.10",
```

Comment:
adding it into this pr, because it doesn't seem to break anything for us
BUT it looks like the mac crashes have disappeared (at least I haven't been able to reproduce them anymore in the last hour or so)